src/bptl/work_units/email/templates/review.txt

```diff
@@ -0,0 +1,6 @@
+{% extends "base.txt" %}
+{% load i18n %}{% block body %}{% if reminder %}HERINNERING: {% endif %}Uw {{ review_type }} is vereist. {% if deadline %}U heeft tot {{ deadline }} om te reageren.{% endif %}
+
+Klik alstublieft <a href="{{ kownslFrontendUrl }}">hier</a>.
```

Comment:
I'm not sure how office 365 treats this (I think GMail does parse it as HTML), but having HTML in a .txt file/template seems odd. The e-mail sending code also sends it as plain text, and has no HTML alternative attached. Either way, I would make sure the full URL is just visible in the mail, and not hidden behind a "hier" text as an element attribute. If clients strip tags, the URL is completely gone and can't even be pasted by recipients.

For future reference, I expect that there will exist some sort of template/"sjabloon" thing that will spit out the actual mail content (probably as HTML) which will facilitate the actual templating. Probably we're already doing too much templating here, and I do want to avoid building an entire mail templating solution in BPTL - that should be another (micro)service that possibly gets called in the process before sending the mail.

Reply:
Sure thing.